ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -125,6 +125,7 @@ def __init__(self):
         self.sad_handle = None
         self.process_id = str(os.getpid())
         self.test_params = testutils.test_params_get()
+        self.check_param('kvm_test', False, required = False)
```

Comment:
nit: spaces around `=` do not match lines below. It is preferred to stick to module style.

Reply:
Interestingly the first half of these `check_param`s do not have the spaces while the second half of them have. I originally put it as the last one and matched the style there. For some reason, I moved it to the first one. 

Anyway, I updated all of them so that the style matches. Thanks for catching this.